pages/linux/udisksctl.md

```diff
@@ -0,0 +1,24 @@
+# udisksctl
+
+> A command-line program used to interact with the udisksd daemon process.
+> More information: <http://storaged.org/doc/udisks2-api/latest/udisksctl.1.html>.
+
+- Show high-level information about disk drives and block devices:
+
+`udisksctl status`
+
+- Show detailed information about a device:
```

Comment:
This command works with specific partitions too. Maybe we should add an extra example for that.

Reply:
Added another sample for `device partition`